sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -110,6 +110,11 @@ __ESIMD_INTRIN __ESIMD_raw_vec_t(T, N)
 }
 #endif // __SYCL_DEVICE_ONLY__
 
+#ifdef __SYCL_DEVICE_ONLY__
+
+// TODO: For Device compilation, change 'dpas_info' function argument
+// (precision, depth, repeat, signedness) into template argument like
+// __esimd_dpas* intrinsic declarations for host mode
```

Comment:
I suggest to remove duplicates of this TODO and change it as follows:
```
// TODO: __esimd_dpas* should have single declaration for host and device:
// void __esimd_dpas*(...)
// #ifdef __SYCL_DEVICE_ONLY__
//  ;
// #else
// { <emu_implemenation_of_intrinsic>; }
// #endif // __SYCL_DEVICE_ONLY__
```

Reply:
Done.